README.md

```diff
@@ -43,7 +43,7 @@ This part is using the snap store in order to have easy way to update all the de
 
 ### Smart device preparations:
 
-Setting up instructions for the new device can be found in the [CyBear Jinni Smart Device Resources](https://github.com/CyBear-Jinni/CBJ_Smart-Device_Resources) repository.
+Setting up instructions for the new device can be found in the [CyBear Jinni Smart Device Resources](https://github.com/CyBear-Jinni/cbj_smart-device_Resources) repository.
```

Comment:
cbj_smart-device_Resources should stay CBJ_Smart-Device_Resources as this repo didn't changed his name

Reply:
Okay let me just change and confirm

Reply:
It is same. It does not changed

Reply:
It should be `CBJ_Smart-Device_Resources`